### PR TITLE
Item likes Implementation

### PR DESCRIPTION
**Pull request by ometman**
I have implemented the item likes.
- It is implemented on feature branch 'item-likes'
- A like button is implemented on homepage besides each item
- Involvement API called to display current item likes 
- Click event is added to the buttons to update the item likes count

### CODE REVIEW

File: src/modules/getLikesData.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const getLikesurl = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/votSVyM8b966LHhFVfLV/likes';
+
+const getLikesData = async () => {
+  const response = await fetch(getLikesurl);
+  const feedback = await response.json();
+  return feedback;
```
Comment (enocol) [inline src/modules/getLikesData.js:6]: Here in this powerful function you wrote, I could see that, the function is fetching a resource from the API, but there is no id in the URL for the **likes** endpoint like this /apps/**:app_id**/likes/ The app_id represents the id of a resource you are fetching from the API. Please kindly have a second look to be sure. That may also be the reason why clicking on each like behaves undesirably like that.

**Review by enocol — CHANGES_REQUESTED** [review]
unexpected
When i click on each like like twice, this is what we get

**Review by enocol — CHANGES_REQUESTED** [review]
fix linters.

File: src/modules/getLikesData.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+};
```
[the remark this answers is not in the document]
Comment (ometman) [reply src/modules/getLikesData.js]: Okay

File: src/modules/getLikes.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import getLikesData from './getLikesData.js';
+
+const getLikes = (theEl) => {
+  theEl.forEach((el, elId) => {
+    const eid = (elId + 1).toString();
+    getLikesData().then((res) => {
+      const itemID = res.find((item) => item.item_id === eid);
+      el.textContent = itemID.likes;
+    });
```
[the remark this answers is not in the document]
Comment (ometman) [reply src/modules/getLikes.js]: Oh I see, that was a response from the promise, appear on line 6 and 7. Thanks 😄

File: src/modules/getLikesData.js (hunk 1 above)
Comment (ometman) [reply src/modules/getLikesData.js]: Thanks. The API has no add ID as that of the reservation and comments. It ends with 'likes' for both POST and GET. 👍

**Pull request merged**